reactivesocket-core/src/test/java/io/reactivesocket/frame/LeaseFrameFlyweightTest.java

```diff
@@ -0,0 +1,19 @@
+package io.reactivesocket.frame;
+
+import org.agrona.concurrent.UnsafeBuffer;
+import org.junit.Test;
+
+import java.nio.ByteBuffer;
+
+import static org.junit.Assert.*;
+
+public class LeaseFrameFlyweightTest {
```

Comment:
Can you add 2 tests 

1) that passes with 2 ^ 24 - 1 - (9 + 4 * 2) bytes worth of metadata
and fails with
2) one byte more